docs/components/Popover/Popover.stories.mdx

```diff
@@ -29,18 +29,76 @@ See
 
 ### Functional
 
+<Banner type="warning" dismissible={false}>
+  A "functional" Popover hasn't been implemented in a componentized fashion yet.
+</Banner>
+
 Reveal a list of available actions to the user. For example, if the user clicks
 on an element and there are four potential actions they might take, Popover is a
 great way to present those actions. If you're looking to provide a menu of
 actions that comes complete with a trigger button,
 [Menu](../?path=/docs/components-navigation-menu--docs) has that bundle ready to
 go.
 
-<Banner type="warning" dismissible={false}>
-  A "functional" Popover hasn't been implemented in a componentized fashion yet.
-</Banner>
+### UNSAFE\_ props (Advanced Usage)
```

Comment:
Used (Advanced Usage) to copy what is happening in List Docs where they use `[Custom Rendering (Advanced Usage)](https://atlantis.getjobber.com/?path=/docs/components-lists-and-tables-list--docs#custom-rendering-advanced-usage)`

Reply:
Feels like this should be an h2 under something like "Component customization", until such time as it can live under its' own tab in the new docs site, rather than nested under design guidance?

Also nit, but we should be [using sentence case](https://atlantis.getjobber.com/content/formatting) (ie `(advanced usage)` instead of `(Advanced Usage)`)

Reply:
Good idea - because then we see "Component customization" in the side rail. Ok I updated that here 99c7f556f63b765aff9934ecb9d8bae4e8b070c9
Added that to `List` docs as well. There will be a time where we have "Custom rendering" and "UNSAFE_ props", both of which should go under "Component customization"

<img width="599" alt="Screenshot 2025-01-09 at 9 02 33 PM" src="https://github.com/user-attachments/assets/54a8e59e-3a0c-4646-8b95-2459d4cf5080" />

Reply:
:noice: